EnglishFilter/sections/general_extensions.txt

```diff
@@ -140,6 +140,8 @@ xxxdan.com#%#AG_abortInlineScript(/nasl/, 'document.cookie');
 ! https://github.com/AdguardTeam/AdguardFilters/issues/23560
 za.gl#%#window.open = function() {};
 za.gl#%#AG_abortInlineScript(/popup/, 'redirect');
+! https://github.com/AdguardTeam/AdguardFilters/issues/26217
+turbobit.cc#%#document.cookie = "BetterJsPop0=1; path=/payments/sms";
```

Comment:
This rule is not required on my end. `BetterJsPop` is blocked by another JS rule.
@BlazDT please take a look.

Reply:
@Alex-302 I can't find any JS rule for `turbobit.cc`, which rule do you mean?
I see JS rules only for `turbobit.net`.

If I go here - http://turbobit.cc/download/free/bsuce94t0if9 I can reproduce a popups/popunders (in second step, "continue to regular download" or something like that).
By the way, this rule should works too:
```
turbobit.cc#%#Object.defineProperty(window, 'BetterJsPop', { get: function() { return { init: function() { } }; } });
```